table/server/master/src/main/java/alluxio/master/table/Database.java

```diff
@@ -349,6 +355,9 @@ private boolean processJournalEntryInternal(Journal.JournalEntry entry,
     if (entry.hasAddTable()) {
       return applyAddTable(context, entry);
     }
+    if (entry.hasAddTablePartitions()) {
```

Comment:
I think `AlluxioCatalog` also needs this: https://github.com/Alluxio/alluxio/blob/master/table/server/master/src/main/java/alluxio/master/table/AlluxioCatalog.java#L390

Reply:
Added, thanks!